### PR TITLE
Mergemodels

### PR DESCRIPTION
**Pull request by sudiptoguha**
*Description of changes:* This is the same as 267.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java [hunk 1]
```diff
@@ -46,6 +50,22 @@ public RandomCutForestState convert(String json, Precision precision) throws IOE
         return convert(forest, precision);
     }
 
+    public Optional<RandomCutForestState> convert(ArrayList<String> jsons, int numberOfTrees, Precision precision)
```
Comment (wnbts) [inline Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java:53]: a javadoc would be useful such as on when the result will be empty.

**Review by wnbts — APPROVED** [review]

File: Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java (hunk 1 above)
Comment (sudiptoguha) [reply Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java:53]: This code is run once and not intended for general use.

Comment (wnbts) [reply Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java:53]: ok. The data type for jsons should be `List<String>`

Comment (sudiptoguha) [reply Java/serialization/src/main/java/com/amazon/randomcutforest/serialize/json/v1/V1JsonToV2StateConverter.java:53]: "not intended for general use" -> this is to spec of what the caller suggested.

**Review by kaituo — APPROVED** [review]

**Pull request merged**